src/classes/storage.ts

```diff
@@ -43,13 +43,23 @@ export class Storage {
 
   /**
    * @description Setter to update the Ground places.
-   * @param {GroundPlaces} groundPlaces - Ground places to store inside the Storage.
+   * @param {GroundPlace[]} groundPlaces - Ground places to store inside the Storage.
    * @returns {void}
    */
   public setGroundPlaces(groundPlaces: GroundPlace[]): void {
     this.groundPlaces = groundPlaces;
   }
 
+  /**
+   * @description Clone the current Ground places stored.
+   * @returns {GroundPlace[]}
+   */
+  public cloneGroundPlaces(): GroundPlace[] {
+    const groundPlaces: GroundPlace[] = this.getGroundPlaces();
+
+    return JSON.parse(JSON.stringify(groundPlaces));
```

Comment:
i dont think you need this method. You can use lodash deepCopy when you need it, directly in your groundPlaces class, without a storage method.

Reply:
I don't know if this is worth to add a new package like `lodash` only to use maybe two or three methods of the package?

Reply:
I personally find it cleaner than a `parse(stringify())`. And you just import the ones you need. Lets wait for @rimiti and @Simobain review

Reply:
Ok, `lodash` provide us small modules that export only one function, I use now `lodash.clonedeep` module